### PR TITLE
feat(message): add message component

### PR DESCRIPTION
**Pull request by mvhk**

### CODE REVIEW

**Comment by udaypydi** [comment]
@mvhk pls update the branch

File: src/uielements/message/message.component.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import classnames from 'classnames';
+
+function Message(props) {
+  const {
+    className, messageText, color, style, rounded,
+  } = props;
+  let messageClass = `bg-${color}-100 border border-${color}-400 text-${color}-700 px-4 py-3 relative`;
+
+  if (rounded) {
+    messageClass += ' rounded-md';
```
Comment (udaypydi) [inline src/uielements/message/message.component.js:12]: use classnames

Comment (udaypydi) [reply src/uielements/message/message.component.js:12]: refer other components

**Comment by mvhk** [comment]
> @mvhk pls update the branch and fix the issues as well. Shields are breaking.

Donw

File: src/uielements/message/message.component.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    <div
+      className={classnames(
+        `${messageClass}`,
+        `${className}`,
+      )}
+      style={style}
+    >
+      <span className="block sm:inline">
+        {messageText}
+        {' '}
```
Comment (udaypydi) [inline src/uielements/message/message.component.js:25]: remove this empty space

File: src/uielements/message/message.component.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  /**
+   * Adds default border radius to button.
+   * The border radius can be modified through custom styles
+   * or through tailwind classname (`rounded`, `rounded-sm`, `rounded-md` etc)
+   */
+  rounded: PropTypes.bool,
+  /**
+   * Message box can have the Child element.
+   */
+  children: PropTypes.string,
```
Comment (udaypydi) [inline src/uielements/message/message.component.js:57]: default prop for children?

File: src/uielements/message/message.component.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+        `${messageClass}`,
+        `${className}`,
+      )}
+      style={style}
+    >
+      <span className="block sm:inline">
+        {messageText}
+        {' '}
+        {props.children}
+      </span>
```
Comment (udaypydi) [inline src/uielements/message/message.component.js:27]: have a close along with `{children}`

**Comment by mvhk** [comment]
> Please make the changes accordingly

Check Now

**Pull request merged**